pallets/ddc-payouts/src/lib.rs

```diff
@@ -187,30 +213,23 @@ pub mod pallet {
 
 	#[pallet::storage]
 	#[pallet::getter(fn debtor_customers)]
-	pub type DebtorCustomers<T: Config> = StorageDoubleMap<
```

Comment:
I can see that we're adding debt to this structure, but how is it updated afterwards? For example the owner top ups the account and continues using the service

Reply:
also, please rename customer_dept to customer_debt, it's a typo probably

Reply:
debt is not managed and definately not here, as it should be delt in the customers pallet when the owner tops up

Reply:
the structure is a mapping between with keys: clusterid + accountid. When topping up the account, it seems to be inefficient to check all existing clusters. 

Maybe it's better to update ledger structure in account directly? 

The solution I envisioned initially, is to depleted the customer balance until it's 0, and add to the negative balance all of the debt. 
Then when we top up, we would cover the debt first, and only then add to the positive balance

Reply:
currently we're only logging it and later think about what to actually do with it - for now there is no clear decision about it. the idea about negative balance is welcome and will be discussed as an alternative